Makefile

```diff
@@ -94,6 +94,10 @@ clean-schema:
 	rm -rf schema.json
 	rm -rf reference-schema.json
 
+clean-languages:
```

Comment:
@saito-hideki With the steps documented here: https://github.com/ansible/awx/pull/4379#issuecomment-516852509, should we still include the new `clean-languages` target? I'm thinking we probably do but I just wanted to confirm.

Reply:
@jakemcdermott thank you for the follow-up!!
Yeah, I think it would be better to remove `*.mo` file which is generated by `make languages` using `make clean-languages`.
Thank you for your help!